CHANGELOG.md

```diff
@@ -27,6 +27,46 @@ CHANGELOG
 Swift 5.3
 ----------
 
+* [SR-11700][]:
```

Comment:
Could you add the link to the SR to the bottom of this file? Otherwise it won’t get linked properly.

Reply:
Done